server/public/model/utils.go

```diff
@@ -531,6 +531,16 @@ func ToJSON(v any) []byte {
 	return b
 }
 
+// FromJSON decodes an arbitrary object
+func ObjectFromJSON(data io.Reader, obj any, maxBytes int64) any {
```

Comment:
Drive by review: Is this a good case for generics?
```suggestion
func ObjectFromJSON[T any](data io.Reader, obj T, maxBytes int64) T {
```

Reply:
Maybe we also need to return an error in this case to differentiate the error case.

Reply:
Yep...never did like no error being returned...reworked for both.